servers/republik/migrations/sqls/20181119124113-cancellations-up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE DOMAIN cancel_category AS TEXT
```

Comment:
Uh! This looks more maintainable than `ENUM`. Intended?

Reply:
that's exactly the reason! A domain can easily be changed, enums not.